.github/workflows/test.yaml

```diff
@@ -10,14 +10,19 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10']
-        django-version: ['3.2', '4.0']
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
+        django-version: ['3.2', '4.0', '4.1']
         postgres-version: ['10', '11', '12', '13']
         exclude:
           - python-version: 3.7
             django-version: 4.0
           - postgres-version: 10
             django-version: 4.0
+          - postgres-version: 10
+            django-version: 4.1
+          # Django 4.1 requires Python >=3.8
```

Comment:
A comment has been added here to describe why this django-version is not compatible with python 3.7. Is this any different from the other excludes that are listed here?

If not, I would suggest to also add comments for the other excludes describing why they are excluded for testing.

Reply:
I've added the comments in ed92632123b6651929a139132422b3669dac0e1e as this isn't any different.